react/Header/UserAccount/UserAccount.js

```diff
@@ -1,15 +1,12 @@
-import styles from './UserAccount.less';
-
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import ScrollLock from 'react-scrolllock';
-
 import ChevronIcon from '../../ChevronIcon/ChevronIcon';
 import ScreenReaderOnly from '../../ScreenReaderOnly/ScreenReaderOnly';
 import UserAccountMenu from '../UserAccountMenu/UserAccountMenu';
 import { AUTHENTICATED, UNAUTHENTICATED, AUTH_PENDING } from '../../private/authStatusTypes';
-
 import smallDeviceOnly from '../../private/smallDeviceOnly';
+import styles from './UserAccount.less';
```

Comment:
Note: I had to move this to the bottom of the import list because of a minor CSS import order issue which caused the user account menu chevron to be (slightly) the wrong size. This style sheet attempts to override styles provided by `ChevronIcon.less`, so it was logically incorrect to include it first.